src/services/publishing/utils/explorer.tsx

```diff
@@ -0,0 +1,197 @@
+import axios, { AxiosResponse } from "axios";
+import { ETHERSCAN_API_KEY } from "../../../config";
+import { NetworkObject } from "../../../types";
+interface ExplorerHeaders {
+  "Content-Type": string;
+}
+
+interface ExploreParams {
+  module: string;
+  action: string;
+  address: string;
+  startblock: string;
+  endblock: string;
+  page: string;
+  offset: string;
+  sort: string;
+  apikey: string;
+}
+
+interface NetworkAPIDetails {
+  apiKey: string;
+  hostname: string;
+}
+
+interface SimplifiedResponse {
+  status: number;
+  reason: string;
+  address: string;
+}
+
+const getHeaders = (): ExplorerHeaders => {
+  const headers = {
+    "Content-Type": "application/json",
+  } as ExplorerHeaders;
+
+  return headers;
+};
+
+const getParams = (contractAddress: string, apiKey: string): ExploreParams => {
+  const headers = {
+    module: "account",
+    action: "txlist",
+    startblock: "0",
+    endblock: "99999999",
+    page: "1",
+    offset: "1",
+    sort: "asc",
+  } as ExploreParams;
+
+  if (apiKey) {
+    headers["apikey"] = apiKey;
+  }
+
+  if (contractAddress) {
+    headers["address"] = contractAddress;
+  }
+
+  return headers;
+};
+
+const getNetworkAPIDetails = (network: NetworkObject): NetworkAPIDetails => {
```

Comment:
```suggestion
const getNetworkApiDetails = (network: NetworkObject): NetworkApiDetails => {
```
naming convention, up for discussion with team